src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs

```diff
@@ -166,7 +168,38 @@ public async Task<bool> UpdateActivitiesAsync(Proto.Activities activities)
                 Event(e, CreateLoggerArgs(EnumLogLevel.Error, $"UpdateActivitiesAsync Failed - {e.Message}"));
                 if (_throwAPIErrors) 
 					 throw; 
-				 return false;
+				return false;
+            }
+        }
+
+        public async Task<bool> UpdateActivityPropertyBagAsync(Guid activityId, OneJsonPatchItems propertyBagUpdates)

```

Comment:
Can you please add a ///summary to public methods?  I think it's visible by and helpful to the API consumers.

Reply:
Added comments.

Reply:
thanks!